frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -131,6 +133,15 @@ export function FeatureFlag({ id }: { id?: string } = {}): JSX.Element {
                             </div>
                         }
                     />
+                    {featureFlag.experiment_set && featureFlag.experiment_set?.length > 0 && (
+                        <AlertMessage type="warning" style={{ margin: '1rem 0' }}>
```

Comment:
We're trying to avoid custom CSS like `style={{ margin: '1rem 0' }}`, so I'd change that to `className="mt mb"` to get the same effect. However that's also a smell, and the `space-y` class on the parent html element might even be better, though may be out of scope.

Better docs for this are coming really soon (see team app east's Q3 design systems goal), though feel free to look at the utility classes in `global.scss` for inspiration.